packages/plugin-vue-jsx/index.js

```diff
@@ -110,6 +113,9 @@ function vueJsxPlugin(options = {}) {
           configFile: false
         })
 
+        // support multi class in jsx #5525
+        result.code = result.code.replace(/(?<=:\s*(["'][^"']*)?)\s*\n/g, ' ')
```

Comment:
The regex doesn't look safe to me.
It may affect other code patterns.

This issue should be fixed in https://github.com/vuejs/jsx-next/ instead

Reply:
but how come vite throwing the error?

![140035689-e413a513-7cca-45f4-8f53-f7ce6ba875d6](https://user-images.githubusercontent.com/77238199/140079977-95767c29-40b9-4be2-8abe-5c0d81032d5f.png)

Reply:
@sodatea hello, just to comment here again in case you didnt notice or forgot. Feel free to reply.

Reply:
The problem here is that the babel JSX plugin doesn't transform newlines in `class` into spaces, so the output JavaScript contains invalid syntax that Vite fails to parse.